platform/os_windows.go

```diff
@@ -190,24 +194,33 @@ func (p *execClient) ExecutePowershellCommand(command string) (string, error) {
 	return strings.TrimSpace(stdout.String()), nil
 }
 
-// SetSdnRemoteArpMacAddress sets the regkey for SDNRemoteArpMacAddress needed for multitenancy
-func SetSdnRemoteArpMacAddress() error {
-	p := NewExecClient(nil)
+// SetSdnRemoteArpMacAddress sets the regkey for SDNRemoteArpMacAddress needed for multitenancy if hns is enabled
+func SetSdnRemoteArpMacAddress(execClient ExecClient) error {
```

Comment:
Just curious to know why did we change the method signature and pass in ExecClient as parameter?

Reply:
passing the execClient to this func so we can pass the mockClient to this func in UT to mock different behaviors